lib/airbrake-ruby/backtrace.rb

```diff
@@ -128,8 +128,9 @@ def self.parse(exception, logger)
     # @param [Exception] exception
     # @return [Boolean]
     def self.java_exception?(exception)
-      defined?(Java::JavaLang::Throwable) &&
-        exception.is_a?(Java::JavaLang::Throwable)
+      (defined?(Java::JavaLang::Throwable) &&
+        exception.is_a?(Java::JavaLang::Throwable)) ||
+        Patterns::JAVA =~ exception.backtrace.first
```

Comment:
Nitpick: Since it's a boolean method, I would do `!!(Patterns::JAVA =~ exception.backtrace.first)`.